src/pages/loginView.js

```diff
@@ -44,6 +44,7 @@ export default function LoginView() {
     }
 
     if (validLogin) {
+      setSuccessfulLogin(true);
```

Comment:
Once this runs, it never flips back, meaning that if login fails for any reason (e.g. incorrect password) the user is locked out unless they refresh the page. Running `setSuccessfulLogin(false)` on line 55 should fix that I think :)